### ISSUE TITLE
Color contrast on footer links is not accessible

### ISSUE THREAD

**Issue by dnoneill** [pod4lib/aggregator#1248]
<img width="1431" height="219" alt="Image" src="https://github.com/user-attachments/assets/324efd0d-be85-4bc7-8b37-4e3dd51fdbfa" />
<img width="816" height="843" alt="Image" src="https://github.com/user-attachments/assets/74f736f2-3e8b-4e90-b3e0-71de736de310" />
